app/src/main/java/com/hapramp/datastore/DataStore.java

```diff
@@ -692,5 +694,29 @@ public void run() {
     }.start();
   }
 
+  public static void performAllCommunitySync() {
+    new DataStore().requestAllCommunities(new CommunitiesCallback() {
+      @Override
+      public void onWhileWeAreFetchingCommunities() {
```

Comment:
I think this should be `onCommunitiesFetching`, based on the other methods form the interface.

Reply:
Ok, this is a matter of naming. I will change it